component/storage/couchdb/store.go

```diff
@@ -181,8 +181,8 @@ func WithLogger(logger logger) Option {
 	}
 }
 
-// PingCouchDB performs a readiness check on the CouchDB instance located at url.
-func PingCouchDB(url string) error {
```

Comment:
Note: CouchDB already had a "ping" method, but it's not quite the same as a regular ping call... so I renamed it and also added a Ping method that works similarly to the other ping implementations you'll see in this PR.